frontend/src/bundles/common/components/sidebar/sidebar.tsx

```diff
@@ -105,6 +106,23 @@ const Sidebar: React.FC<Properties> = ({ children }) => {
                             label="My Avatar"
                         />
                     </Link>
+                    <Text color="background.600" variant="bodySmall">
```

Comment:
With this fontsize the text doesn't fit when sidebar is collaped

Reply:
Fixed